backend/main.go

```diff
@@ -231,17 +242,33 @@ func main() {
 	router := mux.NewRouter()
 	router.HandleFunc(newrelic.WrapHandleFunc(app, "/ws", hub.Handler))
 	router.HandleFunc(newrelic.WrapHandleFunc(app, "/download/{path:.*}", nomad.downloadFile))
-	router.PathPrefix("/static").Handler(http.FileServer(myAssetFS))
 	router.HandleFunc(newrelic.WrapHandleFunc(app, "/config.js", func(w http.ResponseWriter, r *http.Request) {
 		response := make([]string, 0)
 		response = append(response, fmt.Sprintf("window.NOMAD_READ_ONLY=%s", strconv.FormatBool(cfg.ReadOnly)))
 		response = append(response, fmt.Sprintf("window.NOMAD_ADDR=\"%s\"", cfg.Address))
 		response = append(response, fmt.Sprintf("window.NOMAD_LOG_LEVEL=\"%s\"", cfg.LogLevel))
 
+		var endpointURL *url.URL
+		var err error
+		if cfg.ProxyAddress != "" {
+			endpointURL, err = url.Parse(cfg.ProxyAddress)
+			if err != nil {
+				logger.Infof("Error parsing supplied web.proxy-address of %s: %s", cfg.ProxyAddress, err)
```

Comment:
Hmm, so the only issue I see with that is the error handling here is not necessarily validation, but trapping errors that come from `url.Parse`. So if we wanted to validate this for the construction of `NOMAD_ENDPOINT`, we would have to execute this twice, and drop one of the results on the floor. It seems to me that putting this in validation we would have to either build `NOMAD_ADDR` there or test-parse it, and drop the result on the floor. The former seems like it would be polluting the `Config` struct with environment, and the latter seems like a duplication of work. Thoughts?